packages/core/strapi/lib/services/metrics/sender.js

```diff
@@ -44,6 +44,7 @@ module.exports = (strapi) => {
   const anonymousMetadata = {
     environment: strapi.config.environment,
     os: os.type(),
+    osArch: os.arch(),
```

Comment:
@alexandrebodin Fabien suggested in https://github.com/strapi/strapi/pull/13872#issuecomment-1200886065 to add the call here too. I've tried to be consistent with the naming, but I'm wondering why these properties are camelCased here and used with underscores in generators?

//cc @fdel-car